### ISSUE TITLE
Seeing pod count as zero in the UI for mono vertex when the actual pod count is one

### ISSUE THREAD

**Issue by yhl25** [numaproj/numaflow#1959]
![Screenshot 2024-08-19 at 2 16 40 PM](https://github.com/user-attachments/assets/6fc83a79-503b-4280-9d22-25df1b5c2fc8)

**Comment by yhl25** [comment]
Pod count in the UI is extracted from `monovertex.spec.replicas`, but for this mvtx, we don't see the replicas field being set
```json
  "spec": {
      "source": {},
      "sink": {},
      "volumes": [],
      "scale": {}
  }
```

**Comment by whynowy** [comment]
> Pod count in the UI is extracted from `monovertex.spec.replicas`, but for this mvtx, we don't see the replicas field being set
> 
> ```json
>   "spec": {
>       "source": {},
>       "sink": {},
>       "volumes": [],
>       "scale": {}
>   }
> ```

It should be extracted from `status.replicas`

**Comment by vigith** [comment]
Closed via #1965
